src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -0,0 +1,78 @@
+package nextstep.subway.station;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import nextstep.subway.station.dto.StationResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static nextstep.subway.util.StationStep.*;
+
+@DisplayName("지하철역 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class StationAcceptanceTest {
+
+    List<String> STATION_NAME_LIST = List.of("강남역", "역삼역");
+
+    @DisplayName("지하철역을 생성한다.")
+    @Test
+    void createStation() {
+        // when
+        지하철_역_등록("강남역");
+
+        // then
+        List<String> stationNames = 지하철_역_전체_조회();
+        assertThat(stationNames).containsAnyOf("강남역");
+    }
+
+    @DisplayName("2개의 지하철역을 생성한 다음, 2개의 지하철역을 조회한다.")
+    @Test
+    void showStation() {
+        // given
+        for (int stationNameIdx = 0; stationNameIdx < STATION_NAME_LIST.size(); stationNameIdx++) {
+            String stationName = STATION_NAME_LIST.get(stationNameIdx);
+            지하철_역_등록(stationName);
+        }
```

Comment:
```suggestion
        // given
        지하철_역_등록("강남역");
        지하철_역_등록("역삼역");
```

atdd는 개발자가 아닌 인수자가 보는 테스트이죠!? 그렇다면, for문을 쓰는 것보다 가독성을 더 챙겨주는게 좋지 않을까합니다.